### PR TITLE
Publish 1.0.x docs

### PR DESCRIPTION
**Pull request by mdedetrich**

### CODE REVIEW

**Review by nvollmar — APPROVED** [review]
lgtm

File: .github/workflows/publish-1.0-docs.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          cp -r target/scala-2.13/unidoc target/nightly-docs/docs/pekko-connectors-kafka/1.0.0/api
+          cp -r target/scala-2.13/unidoc target/nightly-docs/docs/pekko-connectors-kafka/1.0/api
+          rm -r target/scala-2.13/unidoc
+
+      - name: Upload nightly docs
+        uses: ./.github/actions/sync-nightlies
+        with:
+          upload: true
+          switches: --archive --compress --update --delete --progress --relative
+          local_path: target/nightly-docs/./docs/pekko-connectors-kafka/1.0.0 # The intermediate dot is to show `--relative` which paths to operate on
```
Comment (pjfanning) [inline .github/workflows/publish-1.0-docs.yml:49]: you'll need a 2nd copy of this stage that handles `1.0`

**Review by pjfanning — CHANGES_REQUESTED** [review]

File: .github/workflows/publish-1.0-docs.yml (hunk 1 above)
Comment (mdedetrich) [reply .github/workflows/publish-1.0-docs.yml:49]: Ah I see, thanks doing that now

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**